apps/scratchpad/src/createAccount.ts

```diff
@@ -64,24 +69,52 @@ export async function createAccount() {
     return sigHex;
   };
   const dryRun = false;
+
+  const pubKey = Buffer.from(pubKeyHex.substring(56), "hex");
+  if (pubKey.length !== 64) {
+    throw new Error("Invalid public key, wrong length");
+  }
+
+  const key1 = `0x${pubKey.subarray(0, 32).toString("hex")}` as Hex;
+  const key2 = `0x${pubKey.subarray(32).toString("hex")}` as Hex;
+  const salt = 0n;
+  const args = [[key1, key2], salt] as const;
+
+  const address = await publicClient.readContract({
+    ...accountFactoryConfig,
+    functionName: "getAddress",
+    args,
+  });
```

Comment:
step 1: call RPC to get our own counterfactual address

(we could compute this ourselves. not urgent but a nice future optimization)